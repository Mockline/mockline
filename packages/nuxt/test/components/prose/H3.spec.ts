import { describe, it, expect } from 'vitest'
// @ts-ignore
import { MH3 } from '#components'
import type { TypeOf } from 'zod'
import ComponentRender from "../../component-render";

describe('H3', () => {
  it.each([
    ['basic case', {}]
  ])('renders %s correctly', async (nameOrHtml: string, options: TypeOf<typeof MH3.props>) => {
    const html = await ComponentRender(nameOrHtml, options, MH3)
    expect(html).toMatchSnapshot()
  })
})
