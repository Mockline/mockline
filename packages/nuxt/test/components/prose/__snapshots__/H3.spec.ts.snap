// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`H3 > renders basic case correctly 1`] = `"<h3 class="text-xl font-bold mt-8 mb-3 scroll-mt-[calc(var(--header-height))] [&amp;>a]:focus-visible:outline- [var(--ui-primary)] [&amp;>a>code]:border-dashed hover:[&amp;>a>code]:border-[var(--ui-primary)] hover:[&amp;>a>code]:text-[var(--ui-primary)] [&amp;>a>code]:text-lg/6 [&amp;>a>code]:font-bold"></h3>"`;
